sno/init.py

```diff
@@ -888,7 +965,10 @@ def import_table(
             xml_metadata=info.get('xmlMetadata'),
         )
 
-    structure.fast_import_tables(repo, loaders, message=message, version=version)
+    extra_blobs = []
+    if version == 2:
+        extra_blobs = [(VERSION_BLOB_PATH, b"0.5\n")]
```

Comment:
Feels like we don't want 0.5/0.6/0.12 stored in the repos if the datasets are compatible? 

Ie. I think this should be a version of breaking changes to repository structure — upgrading Sno (with a compatible repo structure) shouldn't mean you have to (or are even encouraged to) commit.

So dataset v1 vs v2 rather than Sno v0.5 vs v0.6.

Reply:
Changed to 1, 2 etc in ".sno-format"